### DISCUSSION TITLE
Обход Ростелеком

### DISCUSSION THREAD

**Discussion by fnDxrk** [bol-van/zapret#473] [Q&A] [answered]
Сегодня перестал работать Discord и Youtube. Я немного исправил свой конфиг, но не работает голосовой чат в Discord (RTC). Пожалуйста, отправьте ваши конфигурации и домены, которые вы используете, если у вас провайдер Ростелеком и всё работает.
Вот параметры, которые я использую
>NFQWS_OPT_DESYNC="--dpi-desync=fake,disorder2  --dpi-desync-repeats=20 --dpi-desync-ttl=12 --dpi-desync-fooling=md5sig --dpi-desync-fake-tls=/opt/zapret/files/fake/tls_clienthello_www_google_com.bin"

>NFQWS_OPT_DESYNC_QUIC="--dpi-desync=fake,disorder2 --dpi-desync-repeats=20 --dpi-desync-fooling=md5sig --dpi-desync-fake-quic=/opt/zapret/files/fake/quic_initial_www_google_com.bin"

**Comment by electrifying** [comment, marked as answer]
скрипт от bol van для дискорда

https://ntc.party/t/zapret-whats-new/61/157

**Reply by fnDxrk** [reply]
Всё работает, самое главное про порты не забыть.
Спасибо большое тебе и автору!

**Reply by dronotte** [reply]
@fnDxrk А дописывал ли что-то сверх скрипта, который bol van написал?
Можешь поделиться конфигом?

**Reply by fnDxrk** [reply]
Нет, только в своём конфиге добавил переменные, как указано в инструкции, и всё.
https://github.com/fnDxrk/dotfiles/blob/main/ArchLinux.md#zapret

**Reply by anton2887** [reply]
> Всё работает, самое главное про порты не забыть. Спасибо большое тебе и автору!

Как открыть порты, подскажите пожалуйста и какие нужно?

**Reply by fnDxrk** [reply]
> Как открыть порты, подскажите пожалуйста и какие нужно?

/opt/zapret/config : 
>#QUIC_PORTS=443,444
>DISCORD_PORTS=50000-65535
>DISCORD_SUBNETS="66.22.196.0/22 66.22.200.0/21 66.22.208.0/20 66.22.224.0/22 66.22.230.0/23 66.22.232.0/21 66.22.240.0/21 66.22.248.0/24"

**Comment by Ensaej** [comment]
спасибо добрый человек, опять РТ сломал ютуб, уже частит, раз в неделю менять конфиг

**Comment by TurboKirill** [comment]
Хорошо, "уважаемый" Роскомнадзор, сейчас отправим))

**Comment by THUGGERRR** [comment]
Рт поменяли мне роутер и после этого у меня перестал работать маджестик сервера гта 5, вот что пишет не удалось подключиться к HTTP серверу, может ктото знает как можно обойти эт без впн или что вообще делать
